contributed_definitions/nyaml/NXpeak.yaml

```diff
@@ -1,136 +1,42 @@
 category: base
 doc: |
-  Description of peaks, their functional form or measured support.
+  Base class for peaks, their functional form, and measured support.
```

Comment:
```suggestion
  Base class for spectral peaks, their functional form, and measured support.
```

Reply:
What does **measured support** mean?

Reply:
Discretization (points) at which the function has been evaluated.

Reply:
Ok, then state this explicitly. How about
"Base class for desribing spectral peaks, their functional form, and support values (i.e., the discretization (points) at which the function has been evaluated)."

Reply:
I support will integrate this  as I will likely be hitting a merge conflict when integrating this into fairmat.